<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Form</class>
 <widget class="QWidget" name="Form">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1181</width>
    <height>789</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>1181</width>
    <height>789</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>1181</width>
    <height>789</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QLabel" name="background">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1181</width>
     <height>782</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="name_lbl">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>15</y>
     <width>271</width>
     <height>61</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font: 75 italic 25pt &quot;MS Sans Serif&quot;;
background-color: rgb(255, 255, 255);
border-radius: 5px;
border: 2px solid #000000;</string>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="alignment">
    <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignTop</set>
   </property>
  </widget>
  <widget class="QLabel" name="main1_lbl_2">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>410</y>
     <width>521</width>
     <height>31</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font: 75 italic 18pt &quot;MS Sans Serif&quot;;
background-color: rgb(255, 255, 255);
border-radius: 5px;
border: 2px solid #000000;</string>
   </property>
   <property name="text">
    <string>Информация для отдыхающих (туристов):</string>
   </property>
   <property name="textFormat">
    <enum>Qt::AutoText</enum>
   </property>
   <property name="alignment">
    <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="lbl7_7">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>520</y>
     <width>221</width>
     <height>16</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font: 10pt &quot;MS Reference Sans Serif&quot;;
background-color: rgb(255, 255, 255);
border-radius: 5px;
border: 2px solid #000000;</string>
   </property>
   <property name="text">
    <string>Телефоны экстренных служб:</string>
   </property>
  </widget>
  <widget class="QLabel" name="main1_lbl">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>90</y>
     <width>361</width>
     <height>31</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font: 75 italic 18pt &quot;MS Sans Serif&quot;;
background-color: rgb(255, 255, 255);
border-radius: 5px;
border: 2px solid #000000;</string>
   </property>
   <property name="text">
    <string>Основные характеристики:</string>
   </property>
   <property name="textFormat">
    <enum>Qt::AutoText</enum>
   </property>
   <property name="alignment">
    <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="lbl6">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>280</y>
     <width>151</width>
     <height>16</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font: 10pt &quot;MS Reference Sans Serif&quot;;
background-color: rgb(255, 255, 255);
border-radius: 5px;
border: 2px solid #000000;</string>
   </property>
   <property name="text">
    <string>Главы государства:</string>
   </property>
  </widget>
  <widget class="QLabel" name="lbl_7">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>310</y>
     <width>271</width>
     <height>16</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font: 10pt &quot;MS Reference Sans Serif&quot;;
background-color: rgb(255, 255, 255);
border-radius: 5px;
border: 2px solid #000000;</string>
   </property>
   <property name="text">
    <string>Где состоит (в каких организациях):</string>
   </property>
  </widget>
  <widget class="QLabel" name="lbl1">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>130</y>
     <width>201</width>
     <height>16</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font: 10pt &quot;MS Reference Sans Serif&quot;;
background-color: rgb(255, 255, 255);
border-radius: 5px;
border: 2px solid #000000;</string>
   </property>
   <property name="text">
    <string>Полное название страны:</string>
   </property>
  </widget>
  <widget class="QLabel" name="flag_lbl">
   <property name="geometry">
    <rect>
     <x>130</x>
     <y>580</y>
     <width>241</width>
     <height>191</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font: 75 italic 18pt &quot;MS Sans Serif&quot;;
border-radius: 5px;
border: 2px solid #000000</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="lbl10">
   <property name="geometry">
    <rect>
     <x>400</x>
     <y>580</y>
     <width>81</width>
     <height>20</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font: 12pt &quot;MS Reference Sans Serif&quot;;
background-color: rgb(255, 255, 255);
border-radius: 5px;
border: 2px solid #000000;</string>
   </property>
   <property name="text">
    <string>Герб: </string>
   </property>
  </widget>
  <widget class="QLabel" name="lbl3">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>190</y>
     <width>81</width>
     <height>16</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font: 10pt &quot;MS Reference Sans Serif&quot;;
background-color: rgb(255, 255, 255);
border-radius: 5px;
border: 2px solid #000000;</string>
   </property>
   <property name="text">
    <string>Площадь: </string>
   </property>
  </widget>
  <widget class="QLabel" name="lbl7_8">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>550</y>
     <width>141</width>
     <height>16</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font: 10pt &quot;MS Reference Sans Serif&quot;;
background-color: rgb(255, 255, 255);
border-radius: 5px;
border: 2px solid #000000;</string>
   </property>
   <property name="text">
    <string>Банковские карты:</string>
   </property>
  </widget>
  <widget class="QLabel" name="lbl5">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>250</y>
     <width>101</width>
     <height>16</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font: 10pt &quot;MS Reference Sans Serif&quot;;
background-color: rgb(255, 255, 255);
border-radius: 5px;
border: 2px solid #000000;</string>
   </property>
   <property name="text">
    <string>Полушария:</string>
   </property>
  </widget>
  <widget class="QLabel" name="lbl11">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>580</y>
     <width>81</width>
     <height>20</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font: 12pt &quot;MS Reference Sans Serif&quot;;
background-color: rgb(255, 255, 255);
border-radius: 5px;
border: 2px solid #000000;</string>
   </property>
   <property name="text">
    <string>Флаг:</string>
   </property>
  </widget>
  <widget class="QLabel" name="lbl4">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>220</y>
     <width>171</width>
     <height>16</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font: 10pt &quot;MS Reference Sans Serif&quot;;
background-color: rgb(255, 255, 255);
border-radius: 5px;
border: 2px solid #000000;</string>
   </property>
   <property name="text">
    <string>Климатические пояса:</string>
   </property>
  </widget>
  <widget class="QLabel" name="lbl8">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>340</y>
     <width>151</width>
     <height>16</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font: 10pt &quot;MS Reference Sans Serif&quot;;
background-color: rgb(255, 255, 255);
border-radius: 5px;
border: 2px solid #000000;</string>
   </property>
   <property name="text">
    <string>Национальный язык:</string>
   </property>
  </widget>
  <widget class="QLabel" name="lbl7_3">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>490</y>
     <width>191</width>
     <height>16</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font: 10pt &quot;MS Reference Sans Serif&quot;;
background-color: rgb(255, 255, 255);
border-radius: 5px;
border: 2px solid #000000;</string>
   </property>
   <property name="text">
    <string>Достопримечательности:</string>
   </property>
  </widget>
  <widget class="QLabel" name="gerb_lbl">
   <property name="geometry">
    <rect>
     <x>510</x>
     <y>580</y>
     <width>241</width>
     <height>191</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font: 75 italic 18pt &quot;MS Sans Serif&quot;;
border-radius: 5px;
border: 2px solid #000000</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="lbl7_2">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>460</y>
     <width>271</width>
     <height>16</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font: 10pt &quot;MS Reference Sans Serif&quot;;
background-color: rgb(255, 255, 255);
border-radius: 5px;
border: 2px solid #000000;</string>
   </property>
   <property name="text">
    <string>Какие реквизиты нужны для въезда:</string>
   </property>
  </widget>
  <widget class="QLabel" name="lbl2">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>160</y>
     <width>181</width>
     <height>16</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font: 10pt &quot;MS Reference Sans Serif&quot;;
background-color: rgb(255, 255, 255);
border-radius: 5px;
border: 2px solid #000000;</string>
   </property>
   <property name="text">
    <string>Численность населения:</string>
   </property>
  </widget>
  <widget class="QLabel" name="lbl9">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>370</y>
     <width>71</width>
     <height>16</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font: 10pt &quot;MS Reference Sans Serif&quot;;
background-color: rgb(255, 255, 255);
border-radius: 5px;
border: 2px solid #000000;</string>
   </property>
   <property name="text">
    <string>Валюта:</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
